.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -36,6 +36,29 @@ jobs:
         run: |
           curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
 
+          # Enable the AKS Mariner Kata host while it's in preview.
+
+          az extension add --name aks-preview
+          az feature register \
+            --namespace "Microsoft.ContainerService" \
+            --name "KataVMIsolationPreview"
```

Comment:
Do we really need to do this?

I mean, can't it be done in the Azure account so we avoid having to do the registration here?

Reply:
Yes that's correct. I don't have access to the Azure subscription yet but if you're able to do that we can remove this part.

Reply:
Done from my side!